### ISSUE TITLE
Wrong icon used for cloudy days or hours

### ISSUE THREAD

**Issue by sonora** [WangDaYeeeeee/GeometricWeather#215]
It seems that fo rdays or hours which say 'Cloudy,' 'Mostly cloudy', 'Mostly cloudy with a shower',  etc. the app always falsely displays the icon for 'Partly cloudy'?

**Comment by papjul** [comment]
After #258 is merged, cloudy will correctly report as cloudy. Mostly cloudy will stay Partly cloudy. Mostly cloudy with a shower will stay Rain. Is that OK for you?

**Comment by sonora** [comment]
I think that's a big improvement, thank you!

Having said that, a lot of information to users is being conveyed just via the icons, so any oversimplification there hurts. It can actually make the display and widgets significantly more useful if more icons of the pool https://developer.accuweather.com/weather-icons were adapted.

**Comment by papjul** [comment]
I believe icons need to be generic enough to be available in all weather providers and have a unique "live" animation.

**Comment by sonora** [comment]
Yes, I see. True, of course. Without having tested it yet, looks like this issue could be closed then, thanks again!
